File: concepts/stack/stack_python/stack.py
class Stack:
  def __init__(self) -> None:
    self.values = list()

  def push(self, value):
    self.values.append(value)

  def pop(self):
    return self.values.pop()

  def is_empty(self):
    return len(self.values) == 0

  def __str__(self) -> str:
    return ','.join(str(v) for v in self.values)

File: concepts/stack/stack_node/Stack.js
class Stack {
  constructor() {
    this.list = [];
  }

  push(val) {
    this.list.push(val);
  }

  pop() {
    return this.list.pop();
  }

  isEmpty() {
    return this.list.length == 0;
  }

  toString() {
    return this.list.join(",");
  }
}

module.exports = Stack;
